yt_dlp/extractor/franceculture.py

```diff
@@ -44,18 +54,42 @@ def _real_extract(self, url):
             ''',
             webpage, 'video data'))
 
-        video_url = video_data.get('data-url') or video_data['data-asset-source']
-        title = video_data.get('data-asset-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
-
+        title = video_data.get('data-emission-title') or video_data.get('data-diffusion-title') or self._og_search_title(webpage)
         description = self._html_search_regex(
             r'(?s)<div[^>]+class="intro"[^>]*>.*?<h2>(.+?)</h2>',
             webpage, 'description', default=None)
+
+        # page has playlist
+        if (re.search(r'<div[^>]+class="[^"]*?podcast-list[^"?]*?"[^>]*>', webpage) is not None):
+            playlist_data = self._search_regex(
+                r'''(?sx)
+                    <div[^>]+class="[^"]*?podcast-list[^"?]*?"[^>]*>
+                    (.*?)
+                    <div[^>]+class="[^"]*?see-more-anchor[^"]*?">
+                ''',
+                webpage, 'playlist data')
+
+            entries = [
+                self.url_result(video_url, None, video_id, video_title)
```

Comment:
if i remember right, the None field is ie_key
use this `FranceCultureIE.ie_key()`

Reply:
This breaks the download: `ERROR: no suitable InfoExtractor for URL`

Reply:
Which extractor does these URLs belong to?

Reply:
to this one, but I'm only trying to download the URLs, not extract anything from them

Reply:
example url?

Reply:
It's in the tests: https://www.franceculture.fr/emissions/hasta-dente

Reply:
This is not what i meant but running the code, i can see that u are extracting this wrong.
Parse `data-diffusion-path` and append it to the site domain and then add the `FranceCultureIE.ie_key()` as Ie key.

Reply:
sorry I don't know how to get the host, also it works like this?

Reply:
`https://www.franceculture.fr` this is host
and it's supposed to work like this.

Reply:
What ideally should happen is
the series/show/playlist extractor just extracts the individual content url that is then passed to the video extractor to further process.
If the video extraction is costly (like it was with old tiktok for example), only then should the playlist extractor return the actual content.

with your current approach, the urls downloaded via playlist are lacking all the extra metadata.

Reply:
Your approach cannot work, there is no separate page for each episode (and so there is no other metadata). `data-diffusion-path` is not a real URL. If you try to append it to `https://www.franceculture.fr` you get a redirect to the main playlist page:

```
curl -v 'https://www.franceculture.fr/emissions/hasta-dente/episode-1-jeudi-vous-avez-dit-bizarre'
...
< HTTP/1.1 307 Temporary Redirect
< Content-Type: text/html; charset=UTF-8
< Content-Length: 191
< Location: /emissions/hasta-dente
```

So I think the current approach is the best one.

Reply:
OK I think I found the correct solution actually, every track has its own metadata now thanks.